azure-storage-common/azure/storage/common/_deserialization.py

```diff
@@ -56,6 +56,7 @@ def _get_download_size(start_range, end_range, resource_size):
     'content-range': (None, 'content_range', _to_str),
     'x-ms-blob-sequence-number': (None, 'page_blob_sequence_number', _to_int),
     'x-ms-blob-committed-block-count': (None, 'append_blob_committed_block_count', _to_int),
+    'x-ms-blob-public-access': (None, 'public_access', _to_str),
```

Comment:
Hi, are you making this change because you wanted the public access info when calling get_container_properties?

Btw there's a [separate API](https://github.com/Azure/azure-storage-python/blob/master/azure-storage-blob/azure/storage/blob/baseblobservice.py#L730) specifically for getting this property. And the deserialization was done [here](https://github.com/Azure/azure-storage-python/blob/master/azure-storage-blob/azure/storage/blob/_deserialization.py#L146).

Reply:
Yes, it doesn't make sense for the container property model to have "public_access" but not be able to deserialize it from the response. 
https://github.com/Azure/azure-storage-python/blob/master/azure-storage-blob/azure/storage/blob/models.py#L48
This results in the property always having the value `None`

Reply:
@williexu That's a good point. Could you please add a test+recording for this change? Thanks!